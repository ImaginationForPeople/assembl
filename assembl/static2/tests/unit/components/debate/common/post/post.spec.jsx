import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import FoldedPost from '../../../../../../js/app/components/debate/common/post/foldedPost';

describe('FoldedPost component', () => {
  it('should render a folded post', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<FoldedPost nbPosts={1} />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
  it('should render several folded posts', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<FoldedPost nbPosts={3} />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});