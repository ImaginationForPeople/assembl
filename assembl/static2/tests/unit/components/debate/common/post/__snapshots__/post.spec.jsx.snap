// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoldedPost component should render a folded post 1`] = `
<Translate
  count={1}
  value="debate.thread.foldedPostLink"
/>
`;

exports[`FoldedPost component should render several folded posts 1`] = `
<Translate
  count={3}
  value="debate.thread.foldedPostLink"
/>
`;
